requirements/requirements.in

```diff
@@ -4,7 +4,7 @@ cython==0.27.3
 restkit==4.2.2
 iso8601==0.1.11
 jsonobject==0.9.1
-jsonobject-couchdbkit==0.9.1
+jsonobject-couchdbkit==0.9.2
```

Comment:
Just curious, what's the difference between ```requirements.in``` and ```requirements.txt```?

Reply:
@nickpell `requirements.txt` is now compiled from `requirements.in`. I'll be giving a short talk about this later this week (Wed @ 10AM ET). See also https://github.com/dimagi/commcare-hq/pull/21251 (and https://github.com/dimagi/commcare-cloud/pull/1915 on commcare-cloud).

Reply:
👍